src/bincode/bincode.zig

```diff
@@ -797,6 +808,21 @@ const ShredType = enum(u8) {
     pub const @"!bincode-config" = ShredTypeConfig();
 };
 
+pub fn testRoundTrip(comptime T: type, test_data: anytype) !void {
+    const allocator = std.testing.allocator;
+    const expected_bytes = test_data.bincode_serialized_bytes;
+    const expected_struct = try test_data.asStruct(allocator);
```

Comment:
why not just pass `expected_struct` and `expected_byte` as paramters instead of enforcing them as fields/methods on an anytype?

Reply:
87ae4a59b84503a0adc38e2b59ccd0fbe2f59322